package.json

```diff
@@ -15,8 +15,9 @@
   "dependencies": {
     "@react-native-community/masked-view": "0.1.6",
     "@react-navigation/native": "^5.1.1",
+    "@react-navigation/routers": "^5.3.0",
```

Comment:
Wondering what this is used for?

I can't find any usage in this pr

Reply:
It's a transitive dependency of react-navigation/native, but i needed it to be the latest version to avoid errors with `expo --web`

```
UltimateApp
└─┬ @react-navigation/native@5.1.4
  └─┬ @react-navigation/core@5.3.2
    └── @react-navigation/routers@5.2.1 
```